action.yaml

```diff
@@ -40,10 +40,16 @@ inputs:
       the current documentation
     required: true
     type: string
+  base_tag_name:
+    description: |
+      The name of the tag that contains the documentation after it has been uploaded to
+      discourse.
+    required: false
+    type: string
+    default: "upload-charm-docs/base-content"
   base_branch:
     description: |
-      The name of the branch that contains the documentation after it has been uploaded to
-      discourse. Defaults to the default branch of the repository.
+      DEPRECATED - here for backwards compatibility. This input is no longer used.
```

Comment:
I think we can drop it given the current adoption

Reply:
The reason I left it in is because our `operator-workflows` use the input for now. If the input is removed in this PR, as soon as it is merged `operator-workflows` will start to fail. Keeping it in for a short time allows us to have `operator-workflows` keep working and be able to switch and then remove the input in a follow up PR. What do you think?